magento/magento2
# Move blank theme dependencies out of Magento_Theme requirejs-config

### Description

Creating a new theme that does not fallback to the blank theme causes two files to 404 because Magento_Theme module depends on blank theme assets `js/theme.js` and `js/responsive.js`

### Fixed Issues (if relevant)

1. #3784: js/theme & js/responsive files in wrong requirejs-config file?

### Reproduce bug:

1. Follow the steps to create a new theme from the [devdocs](http://devdocs.magento.com/guides/v2.1/frontend-dev-guide/themes/theme-create.html)
2. Don't define a parent theme:
```
<theme xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Config/etc/theme.xsd">
     <title>New theme</title>
 </theme>
```
3. Clear cache and go to the frontend to see the 404s

### Manual testing steps:
1. New theme should no longer attempt to load `theme.js` or `responsive.js`
2. Switch to blank theme and clear cache. Storefront should load `theme.js` and `responsive.js`
